packages/eds-core-react/src/components/Radio/Radio.stories.tsx

```diff
@@ -40,6 +38,8 @@ export const SingleRadio: Story<RadioProps> = () => {
     </div>
   )
 }
+SingleRadio.storyName = 'Single radio buttons'
```

Comment:
Suggest dropping using `storyName` now as it serves no purpose since we are using mdx.